src/sqlacodegen/generators.py

```diff
@@ -514,12 +520,16 @@ def render_column_type(self, coltype: object) -> str:
 
             value = getattr(coltype, param.name, missing)
             default = defaults.get(param.name, missing)
+            if isinstance(value, TextClause):
+                rendered_value = render_callable("text", repr(value.text))
+            else:
+                rendered_value = repr(value)
```

Comment:
Add a blank line to distinguish between the two unrelated if...else blocks.
```suggestion

                rendered_value = repr(value)
```